docs/excel/excel-add-ins-worksheets.md

```diff
@@ -326,6 +326,52 @@ Excel.run(function (context) {
 > - For examples that show how to get ranges from a `Table` object, see [Work with tables using the Excel JavaScript API](excel-add-ins-tables.md).
 > - For examples that show how to search a large range for multiple sub-ranges based on cell characteristics, see [Work with multiple ranges simultaneously in Excel add-ins](excel-add-ins-multiple-ranges.md).
 
+## Filter data
+
+> [!NOTE]
+> `AutoFilter` is currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+An [AutoFilter]((/javascript/api/excel/excel.autofilter) applies data filters across a range within the worksheet. This is created with `Worksheet.autoFilter.apply`, which has the following parameters:
+
+- `range`: The range, specified as either a `Range` object or a string, to which the filter is applied.
+- `columnIndex`: The zero-based column index against which the filter criteria is evaluated.
+- `criteria`: A [FilterCriteria](/javascript/api/excel/excel.filtercriteria) object determining which rows should be filtered based on the column's cell.
+
+The first code sample shows how to add a filter to the worksheet's used range. This filter will hide entries that are not in the top 25%, based on the values in column **3**.
+
+```js
+Excel.run(function (context) {
+    var sheet = context.workbook.worksheets.getActiveWorksheet();
+    var farmData = sheet.getUsedRange();
+
+    // This filter will only show the rows with the top 25% of values in column 3.
+    sheet.autoFilter.apply(farmData, 3, { criterion1: "25", filterOn: Excel.FilterOn.topPercent });
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+The next code sample shows how to refresh the auto-filter using the `reapply` method. This should be done when the data in the range changes.
+
+```js
+Excel.run(function (context) {
+    var sheet = context.workbook.worksheets.getActiveWorksheet();
+    sheet.autoFilter.reapply();
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+The final auto-filter code sample shows how to remove the filters from the worksheet with the `remove` method.
+
+```js
+Excel.run(function (context) {
+    var sheet = context.workbook.worksheets.getActiveWorksheet();
+    sheet.autoFilter.remove();
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+An `AutoFilter` can also be applied to individual tables. See the ["AutoFilter" section of Work with tables using the Excel JavaScript API](excel-add-ins-tables.md#autofilter) for more information.
```

Comment:
Since the hyperlink points to the section, you might consider removing the text `"AutoFilter data" section of "` from the link text (for brevity).

-> See [Work with tables using the Excel JavaScript API](excel-add-ins-tables.md#autofilter) for more information.